Dockerfile

```diff
@@ -1,7 +1,19 @@
-FROM golang
+FROM golang as builder
 
 COPY . /usr/src/lokomotive
 
 WORKDIR /usr/src/lokomotive
 
 RUN make MOD=vendor install-slim
+
+RUN make MOD=vendor install-controller
+
+# Admission webhook
+
+FROM scratch
+
+COPY --from=builder /usr/src/lokomotive /usr/src/lokomotive-controller/
+
+WORKDIR /usr/src/lokomotive-controller
+
+ENTRYPOINT ["./lokomotive-controller"]
```

Comment:
are we keeping the name of the binary as `lokomotive-controller` ?
Perhaps a name that is in line with the the functionality.
@invidian